utils/tests_fetcher.py

```diff
@@ -1064,10 +1064,17 @@ def infer_tests_to_run(
     impacted_files = sorted(set(impacted_files))
     print(f"\n### IMPACTED FILES ###\n{_print_list(impacted_files)}")
 
+    model_impacted = sorted({"/".join(x.split("/")[:3]) for x in impacted_files if x.startswith("tests/models/")})
+
     # Grab the corresponding test files:
     if any(x in modified_files for x in ["setup.py", ".circleci/create_circleci_config.py"]):
         test_files_to_run = ["tests", "examples"]
         repo_utils_launch = True
+    # The number `30` is just a heuristic to determine if we `guess` all models are impacted.
+    elif not filter_models and len(model_impacted) > 30:
```

Comment:
Can we set this as a global param e.g. `THRESH_NO_MODELS` or as CLI flag so we can easily update the message and value + see that it's an arbitrary flag?

Reply:
ok